flink-filesystems/flink-oss-fs-hadoop/src/main/resources/NOTICE

```diff
@@ -0,0 +1,205 @@
+flink-oos-fs-hadoop
+Copyright 2014-2018 The Apache Software Foundation
+
+This project includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.aliyun.oss:aliyun-sdk-oss:3.1.0
+- com.aliyun:aliyun-java-sdk-core:3.4.0
+- com.aliyun:aliyun-java-sdk-ecs:4.2.0
+- com.aliyun:aliyun-java-sdk-ram:3.0.0
+- com.aliyun:aliyun-java-sdk-sts:3.0.0
+- com.fasterxml.jackson.core:jackson-annotations:2.7.0
+- com.fasterxml.jackson.core:jackson-core:2.7.8
+- com.fasterxml.jackson.core:jackson-databind:2.7.8
+- com.fasterxml.woodstox:woodstox-core:5.0.3
+- com.github.stephenc.jcip:jcip-annotations:1.0-1
+- com.google.code.gson:gson:2.2.4
+- com.google.guava:guava:11.0.2
+- com.nimbusds:nimbus-jose-jwt:4.41.1
+- commons-beanutils:commons-beanutils:1.9.3
+- commons-cli:commons-cli:1.3.1
+- commons-codec:commons-codec:1.10
+- commons-collections:commons-collections:3.2.2
+- commons-io:commons-io:2.4
+- commons-lang:commons-lang:2.6
+- commons-logging:commons-logging:1.1.3
+- commons-net:commons-net:3.6
+- net.bytebuddy:byte-buddy-agent:1.8.15
```

Comment:
this is a test dependency, hence not bundled in the jar, hence doesn't have to be listed in NOTICE